### PR TITLE
#1379 Do not run frontend e2e tests if create services failed

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1379 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/scripts/startAndTest.js [hunk 1]
```diff
@@ -18,64 +18,80 @@
...
-  const killSubProcess = () => {
-    debug('========= Kill all subProcess =========');
-    if (cypress) cypress.kill();
-    if (client) client.kill();
-    if (server) server.kill();
-  };
 
-  // Start server
-  debug('========= Start server =========');
+  const defaultEnv = getDefaultEnv();
```
Comment (konekoya) [inline ohara-manager/scripts/startAndTest.js]: @wu87988622 I've tweaked the script a bit so we don't parse the JSON file multiple times (which is usually very expansive) in order to get these env vars.

**Comment by chia7712** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
CI fail build is not related

**Pull request merged**